src/main/java/sparksoniq/jsoniq/compiler/JsoniqExpressionTreeVisitor.java

```diff
@@ -111,7 +110,6 @@
     private Expression otherCurrentExpression;
 
     private PrimaryExpression currentPrimaryExpression;
-    private PostfixExtension currentPostFixExtension;
```

Comment:
I am very glad that number of these special variables in the state of the class has decreased.